hack/build/run-functional-tests.sh

```diff
@@ -38,8 +38,16 @@ arg_oc="${KUBECTL:+-oc-path=$KUBECTL}"
 test_args="${test_args} -ginkgo.v ${arg_master} ${arg_namespace} ${arg_kubeconfig} ${arg_kubectl} ${arg_oc}"
 
 echo 'Wait until all CDI Pods are ready'
-while [ -n "$(./cluster/kubectl.sh get pods -n $CDI_NAMESPACE -o'custom-columns=status:status.containerStatuses[*].ready' --no-headers | grep false)" ]; do
-    sleep 5
+retry_counter=0
+while [ $retry_counter -lt 30 ] && [ -n "$(./cluster/kubectl.sh get pods -n $CDI_NAMESPACE -o'custom-columns=status:status.containerStatuses[*].ready' --no-headers | grep false)" ]; do
+	retry_counter=$((retry_counter + 1))
+    sleep 10
 done
 
+if [ $retry_counter eq 30 ]; then
+	echo "Not all CDI pods became ready"
+	./cluster/kubectl.sh get pods -n $CDI_NAMESPACE
+	exit 1
+git adfi
```

Comment:
?

Reply:
waiting to merge when previous review comments are addressed...

Reply:
Yeah I am not sure what that is, should be fi....